python/ray/experimental/state/state_cli.py

```diff
@@ -292,10 +292,17 @@ def format_get_api_output(
 ) -> str:
     if not state_data or isinstance(state_data, list) and len(state_data) == 0:
         return f"Resource with id={id} not found in the cluster."
+
     if not isinstance(state_data, list):
         state_data = [state_data]
-    state_data = [dataclasses.asdict(state) for state in state_data]
-    return output_with_format(state_data, schema=schema, format=format, detail=True)
+
+    human_readable_state_data = [
+        dataclasses.asdict(schema.humanify(state)) for state in state_data
+    ]
+
+    return output_with_format(
```

Comment:
Sure, I wasn't certain if we wanted to humanify ALL state_data that calls `output_with_format`, but if that's the case, then it would be cleaner if we did the humanify logic in output_with_format.

Reply:
Hmm, I thought I reverted this change already? Perhaps I forgot to push. Let me try again.